cmake/CorrosionConfig.cmake.in

```diff
@@ -6,7 +6,7 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${PACKAGE_PREFIX_DIR}/@CORROSION_INSTALL_PREFIX@@CMAKE_INSTALL_DATADIR@/cmake")
 
-add_executable(Corrosion::Generator IMPORTED GLOBAL)
+add_executable(Corrosion::Generator IMPORTED)
```

Comment:
I think the idiomatic way is typically to keep the executable as a global target (why localise it?), but guard it with an `if (NOT TARGET Corrosion::Generator)`, like in commit 46d715d64165037cb740181f379088a0f077dbba.

That would span all the way until line 19/20 I think.

Reply:
Without the GLOBAL this target has directory scope. I doubt that is enough: You probably do want to use the Corrosion::Generator somewhere outside of this directory (and maybe below).

Reply:
@tronical seems like a much better fix which works for me, thank you.